### PR TITLE
fix namespace issue

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: R/reexports.R [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#' @export
+#' @importFrom effectsize standardize
+effectsize::standardize
```
Comment (DominiqueMakowski) [inline R/reexports.R:3]: can't we import them directly from where they are?

Comment (DominiqueMakowski) [reply R/reexports.R:3]: Oh you mean this is so that users can use standardize() directly without library(effectsize) is it?

**Comment by IndrajeetPatil** [comment]
You can ignore this. I was just trying to check if this solves the check failures in the master, but it doesn't.

**Comment by DominiqueMakowski** [comment]
but what's going on with this standardzie thing is it because of datawizard?

**Comment by IndrajeetPatil** [comment]
No, nothing to do with `datawizard` since it is currently not being used anywhere in the `easystats` ecosystem.

**Comment by IndrajeetPatil** [comment]
> but what's going on with this standardzie thing

Yeah, that's what I am also trying to figure out.

File: R/reexports.R (hunk 1 above)
Comment (strengejacke) [reply R/reexports.R:3]: yes, it's a re-export

Comment (IndrajeetPatil) [reply R/reexports.R:3]: Huh, this was the solution but somehow the PR builds still failed. Anyway, doesn’t matter. Seems to be fixed in the master by now.

**Pull request closed without merging**